### ISSUE TITLE
Views' Tab Title (a.k.a Part title font?) font size is too big and not adjustable.

### ISSUE THREAD

**Issue by AlphaCactus** [aptana/studio3#235]
The font size in tabs was increased and there doesn't appear to be a way to reduce it.  Not even sure where to look in css files/etc for a place to change it. 
It makes it very difficult to switch between open files with a font so large because I can see so little of the file name.  In the image below, none of these tabs is the same filename, but I can't tell.

This problem appeared for me with 3.6.0.201407100658
From the description in
Window -> Preferences -> General -> Appearance -> Colors and Fonts -> View and Editor Folders -> Part Title Font
it seems that this setting should apply, but changing it doesn't appear to do anything even after a restart.
If it matters, this occurs on Windows 7 and 8.1, and the Control Panel -> Display -> "Make it easier to read what's on your screen" setting is set to Smaller - 100% (default).

![tabfontsize2](https://cloud.githubusercontent.com/assets/1965851/3912240/04f49bde-2329-11e4-8d96-a85061267f9a.png)

**Comment by ClemensSchneider** [comment]
I can confirm this issue, too: Aptana version  3.6.1.201410201044

**Comment by nachouve** [comment]
I happens on Linux Mint 17 also.

**Comment by dsmithhayes** [comment]
Same issues occurs: Xubuntu 14.04 - Aptana Studio 3 build: 3.6.1.201410201044

**Comment by jarnix** [comment]
I have the same issue in Windows 10, tried to play with the css files but cannot find the right one or the right setting, and of course, it should not be the solution...

**Comment by dnwx** [comment]
I have same problem but I was managed to solve this problem. The fastest and simplest solution to this problem is by installing CSS File Editor from kepler update site. You could find the menu under help, select kepler update site, when list came up select CSS File Editor.

After restart, go to windows > preferences > general > appearance. You will find an Editor, find CTabItem, append font-size property to it. Click Apply or OK then restart Aptana Studio.

Hopefully, this bug will be fixed without installing CSS File Editor.

**Comment by mediagirl** [comment]
I'm not finding an Eclipse Marketplace plugin named "CSS File Editor". Is it named "Lightweight CSS Editor" instead? Can you post a link to the one you installed?

**Comment by dnwx** [comment]
Yes, that's correct. I've updated my comment from "CSS File Editor" to "Lightweight CSS Editor".

**Comment by mediagirl** [comment]
Thank you for clarifying... With Aptana Studio 3.6.1.201506021939, I'm able to install Eclipse Marketplace, can install Lightweight CSS Editor but I do not see the editor appear after a restart under: windows > preferences > general > appearance. It showed right away for you? What Aptana version are you running?

**Comment by dnwx** [comment]
Yes, it was showed right away in Aptana Studio 3.6.1.201410201044 on Windows 10 and Aptana Studio 3.6.1.201506021939 on Debian Jessie KDE Plasma, bit slow on this one. This following image is an example what appear on Aptana Studio on KDE, I was set CTabItem font-size to 8.

Anyway, this just workaround. Originally, I was installing Lightweight CSS Editor on stable release.

![example](https://cloud.githubusercontent.com/assets/13613205/10036483/a9c8b332-61d2-11e5-9c74-c15cd513b0e8.jpeg)

**Comment by mediagirl** [comment]
I'd guess I'm hitting some bug on Windows 7. When I load that "Appearance" view in preferences, I see this error. Shrug. 

![2015-09-23_1232](https://cloud.githubusercontent.com/assets/249311/10054596/3db3c7aa-61ef-11e5-8c3d-6efd8c0973d1.png)

**Comment by antonio-malcolm** [comment]
A year after reported, and I"m still seeing this issue. The tab text is big, it's ugly, and it takes precious vertical screen real estate, when most laptops are pushing a 16:9 aspect ratio. Any plans to fix this?

**Comment by s101d1** [comment]
Alternative solution is just change the theme to Classic or Default Theme (window -> Preferences -> Appearance). Avoid Light Studio and Dark Studio theme. If you're on GTK theme, open `<aptana_folder>/plugins/org.eclipse.platform_4.3.2.v20140221-1700/css/e4_default_gtk.css`, find `.MPartStack` class, and change the `font-size` there to 9 or whatever size you want, then restart aptana. Modifying e4_basestyle.css or e4_default.css doesn't work for Light/Dark Studio theme unfortunately.
